glassesHud

```diff
@@ -6,60 +6,54 @@ local boxBgColor = 0xFFFFFF
 local boxTextColor = 0xF5F5F5
 
 function drawBox(height)
-  glasses.addBox(1, 1, 155, height, boxBgColor, .2)
+  glasses.addBox(1, 1, 128, height, boxBgColor, .2)
 end
 
 function display(text)
  glasses.clear()
 
  local yPos = 5
  for index,line in utils.orderTable(text) do
-   glasses.addText(5, yPos, line, boxTextColor)
+   glasses.addText(5, yPos, index .. " " .. line, boxTextColor)
```

Comment:
Can you put a : or some other delimiter besides space to separate this between `index` and `line`?

Reply:
Colon was in MC code, missed it in version control.